<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTable</class>
 <widget class="QFrame" name="DataTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="PWMName">
     <property name="text">
      <string>PWMName</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="CompassName">
     <property name="text">
      <string>CompassName</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="GyroName">
     <property name="text">
      <string>GyroName</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PositionX">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="PositionName">
     <property name="text">
      <string>PositionName</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="EncoderName">
     <property name="text">
      <string>EncoderName</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PositionY">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Left:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PWMA">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_4" native="true">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Right:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PWMB">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_5" native="true">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Left:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EncoderA">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>cm/s</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QWidget" name="widget_6" native="true">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Right:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EncoderB">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>cm/s</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QWidget" name="widget_7" native="true">
     <widget class="QLineEdit" name="GyroData">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DegreePerSec">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QWidget" name="widget_8" native="true">
     <widget class="QLineEdit" name="CompassData">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Degree">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
